guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -122,36 +122,36 @@ Flush cache types if you've already tried cleaning the cache and you're still ha
 
 Command usage:
 
-```terminal
-bin/magento cache:clean [type] ... [type]
-bin/magento cache:flush [type] ... [type]
+```bash
+   bin/magento cache:clean [type] ... [type]
```

Comment:
Break lines 126 and 127 into separate command line examples.

Reply:
Done.